.travis.yml

```diff
@@ -6,21 +6,21 @@ node_js:
 sudo: false
 
 cache:
-  directories:
-    - $HOME/.npm
+  yarn: true
 
 matrix:
   fast_finish: true
   allow_failures:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - npm config set spin false
-  - npm install phantomjs-prebuilt
-  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn global add bower phantomjs-prebuilt
+  - phantomjs --version
 
 install:
-  - npm install
+  - yarn install --no-lockfile
```

Comment:
I think we can set the `--non-interactive` flag here as well to avoid command line prompts for e.g. package version ambiguities: https://github.com/yarnpkg/yarn/pull/2770